index 1fff1aa..81169ed 100644
--- a/10-kubeadm.conf
+++ b/etc/systemd/system/kubelet.service.d/10-kubeadm.conf
@@ -4,6 +4,6 @@ Wants=docker.socket

 [Service]
 ExecStart=
-ExecStart=/usr/bin/kubelet --cluster-dns=10.96.0.10 --cluster-domain=cluster.local --authorization-mode=Webhook --kubeconfig=/etc/kubernetes/kubelet.conf --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true --container-runtime=docker --fail-swap-on=false --hostname-override=minikube --client-ca-file=/var/lib/minikube/certs/ca.crt --cgroup-driver=cgroupfs
+ExecStart=/usr/bin/kubelet --cluster-dns=10.96.0.10 --cluster-domain=cluster.local --authorization-mode=Webhook --kubeconfig=/etc/kubernetes/kubelet.conf --bootstrap-kubeconfig=/etc/kubernetes/bootstrap-kubelet.conf --pod-manifest-path=/etc/kubernetes/manifests --allow-privileged=true --container-runtime=docker --fail-swap-on=false --hostname-override=minikube --client-ca-file=/var/lib/minikube/certs/ca.crt --cgroup-driver=cgroupfs --allow-privileged=true --feature-gates=VolumeSnapshotDataSource=true,KubeletPluginsWatcher=true,CSINodeInfo=true,CSIDriverRegistry=true,BlockVolume=true,CSIBlockVolume=true

 [Install]